notebooks/en/structured_generation.ipynb

```diff
@@ -0,0 +1,600 @@
+{
```

Comment:
<p>"The output of the LLM is a string representation of a <em>dictionary,</em> so let's just load it as a <em>dictionary</em> using <code>literal_eval</code>."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>